src/ontology/Modules/occo_terms.owl

```diff
@@ -464,6 +513,249 @@
+        <obo:IAO_0000117>Sam Smith</obo:IAO_0000117>
+        <obo:IAO_0000119>https://storage.googleapis.com/wzukusers/user-12947767/documents/2c18c4f456ce4f6eab176c0983f75791/FOIS_2023_paper_920.pdf</obo:IAO_0000119>
+        <obo:IAO_0000119>https://www.degruyter.com/document/doi/10.1515/jso-2019-0034/html</obo:IAO_0000119>
+        <obo:IAO_0000233>https://github.com/Occupation-Ontology/OccO/issues/31</obo:IAO_0000233>
+        <rdfs:label>credential</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/OCCO_00000119 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OCCO_00000119">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OCCO_00000118"/>
+        <obo:IAO_0000115>A crediential that is associated with an occupation.</obo:IAO_0000115>
+        <obo:IAO_0000117>Bill Duncan</obo:IAO_0000117>
+        <obo:IAO_0000117>Jie Zheng</obo:IAO_0000117>
+        <obo:IAO_0000117>John Beverley</obo:IAO_0000117>
+        <obo:IAO_0000117>Matthew Diller</obo:IAO_0000117>
+        <obo:IAO_0000117>Oliver He</obo:IAO_0000117>
+        <obo:IAO_0000117>Robin McGill</obo:IAO_0000117>
+        <obo:IAO_0000117>Sam Smith</obo:IAO_0000117>
+        <obo:IAO_0000119>https://storage.googleapis.com/wzukusers/user-12947767/documents/2c18c4f456ce4f6eab176c0983f75791/FOIS_2023_paper_920.pdf</obo:IAO_0000119>
+        <obo:IAO_0000119>https://www.degruyter.com/document/doi/10.1515/jso-2019-0034/html</obo:IAO_0000119>
+        <rdfs:label>occupation credential</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/OCCO_00000120 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OCCO_00000120">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0000245"/>
+        <obo:IAO_0000115>An organization that bears a credential grantor role. </obo:IAO_0000115>
+        <obo:IAO_0000117>Bill Duncan</obo:IAO_0000117>
+        <obo:IAO_0000117>Bill Hogan</obo:IAO_0000117>
+        <obo:IAO_0000117>Finn Wilson</obo:IAO_0000117>
+        <obo:IAO_0000117>Giacomo De Colle</obo:IAO_0000117>
+        <obo:IAO_0000117>Jie Zheng</obo:IAO_0000117>
+        <obo:IAO_0000117>John Beverley</obo:IAO_0000117>
+        <obo:IAO_0000117>Matthew Diller</obo:IAO_0000117>
+        <obo:IAO_0000117>Oliver He</obo:IAO_0000117>
+        <obo:IAO_0000117>Robin McGill</obo:IAO_0000117>
+        <obo:IAO_0000117>Sam Smith</obo:IAO_0000117>
+        <rdfs:label>credential granting agency</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/OCCO_00000121 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OCCO_00000121">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/BFO_0000016"/>
+        <obo:IAO_0000115>A disposition borne by an agent in virtue of training such that, if realized, is realized in the successful performance of a skilled task for which that training was pursued.</obo:IAO_0000115>
```

Comment:
I don't see this class in the submitted paper

Reply:
Do you mean 'competence'? I see that in the paper. Though the definition given here needs to be adjusted slightly.